harc_game_web/apps/tasks/fixtures/example_db.json

```diff
@@ -4,7 +4,7 @@
     "pk": 1,
     "fields": {
       "name": "Zadanie 1",
-      "description": "Opis zadania",
+      "description": "To zadanie jest udokumentowane przez Druzynowego 1 i zaakceptowane pzrez Druzynowego 3",
```

Comment:
to jest opis zadania-zadania, a nie wykonanego zadania :) czyli np. "zrób 10 pompek śpiewając odę do radości"

Reply:
Dodałem te opisy, by wiedzieć w jakim stanie są te przykładowe zadania. Oczywiście to nie ma sensu dla użytkowników, ale to tylko dane testowe.